client/lib/support/support-user/localstorage-bypass.js

```diff
@@ -84,11 +84,11 @@ export default function( allowedKeys ) {
 		const _getItem = window.Storage.prototype.getItem.bind( window.localStorage );
 		const _removeItem = window.Storage.prototype.removeItem.bind( window.localStorage );
 
-		Object.defineProperty( Storage.prototype, 'length', {
```

Comment:
Note that Storage is used as an interface for session and local storage. 
https://developer.mozilla.org/en-US/docs/Web/API/Storage

No other place explicitly uses session storage except for social login. (Everything else should be using localForage and whatever compatible driver the user's browser has)